build.sbt

```diff
@@ -94,32 +68,18 @@ val munitVersion = "0.7.29"
 val munitCEVersion = "1.0.7"
 
 lazy val commonSettings = Seq(
-  crossScalaVersions := Seq(Scala3, Scala213),
-  scalacOptions ++= {
-    if (isDotty.value && githubIsWorkflowBuild.value)
-      Seq("-Xfatal-warnings")
-    else
-      Seq.empty
-  }
+  crossScalaVersions := Seq(Scala3, Scala213)
 )
 
 lazy val root =
-  project
-    .in(file("."))
-    .aggregate(
-      core.js,
-      core.jvm,
-      lambda.js,
-      lambda.jvm,
-      sbtLambda,
-      lambdaHttp4s.js,
-      lambdaHttp4s.jvm,
-      lambdaCloudFormationCustomResource.js,
-      lambdaCloudFormationCustomResource.jvm,
-      examples.js,
-      examples.jvm
-    )
-    .enablePlugins(NoPublishPlugin)
+  tlCrossRootProject.aggregate(
+    core,
+    lambda,
+    sbtLambda,
+    lambdaHttp4s,
+    lambdaCloudFormationCustomResource,
+    examples
+  )
```

Comment:
Not only does this cut down the boilerplate, it also gets CI split into JVM and JS jobs for free.